### PR TITLE
fix: use `display: none` to hide closed expendable items

### PR DESCRIPTION
**Pull request by mstruebing**
We tried to use a more accessible form of hiding elements but that doesn't work out as easy as thought with our current integration tests. 
For the ease of implementation I decided to bring `display: none` back for now to be able to finish: https://jira.d2iq.com/browse/D2IQ-65002

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [ ] Info for applicable sections above is provided

### CODE REVIEW

**Review by vacas5 — DISMISSED** [review]

**Comment by vacas5** [comment]
got to do what we got to do, we can revisit this issue later

**Review by vacas5 — APPROVED** [review]

**Comment by vacas5** [comment]
I'd argue that this commit should be a `chore` because nothing actually changes, just the way we go about making something shown/hidden

**Comment by TattdCodeMonkey** [comment]
@vacas5 if its a chore it won't trigger a release when you merge it. so you would have to wait for another PR to integration this change into any apps.

File: packages/fullscreenView/components/FullscrenView.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 import { cx } from "emotion";
+// TODO: This component contains `willReceiveProps`
```
[the remark this answers is not in the document]
Comment (mstruebing) [reply packages/fullscreenView/components/FullscrenView.tsx:3]: We need to have a look at the component and what it does exactly, if an update already fixes the usage of `componentWillReceiveProps` or if we need to use a different package or roll our own component to do that. Similiar to react-toggled

**Comment by vacas5** [comment]
I'm coming in hot with a feature, so may not be an issue. But, it _is_ semantic release right? :)

**Pull request merged**